draft-ietf-ppm-dap.md

```diff
@@ -2436,195 +2436,55 @@ suite:
 
 # Security Considerations {#sec-considerations}
 
-DAP assumes an active attacker that controls the network and has the ability to
-statically corrupt any number of Clients, Aggregators, and Collectors. That is,
-the attacker can learn the secret state of any party prior to the start of its
-attack. For example, it may coerce a Client into providing malicious input
-shares for aggregation or coerce an Aggregator into diverting from the protocol
-specified (e.g., by divulging its input shares to the attacker).
-
-In the presence of this adversary, DAP aims to achieve the privacy and
-robustness security goals described in {{Section 9 of !VDAF}}. Even if DAP
-achieves those goals, there are still some threats it does not defend against:
-
-1. Even benign collect requests may leak information beyond what one might
-   expect intuitively. For example, the Poplar1 VDAF {{!VDAF}} can be used to
-   compute the set of heavy hitters among a set of arbitrary bit strings
-   uploaded by Clients. This requires multiple evaluations of the VDAF, the
-   results of which reveal information to the Aggregators and Collector beyond
-   what follows from the heavy hitters themselves. Or the result of the Prio3Sum
-   VDAF could leak information about outlier values. Note that this leakage can
-   be mitigated using differential privacy ({{dp}}).
-1. On its own, DAP does not defend against Sybil attacks. See {{sybil}} for
-   discussion and potential mitigations.
-
-## Threat model
-
-In this section, we enumerate the actors participating in a Distributed
-Aggregation Protocol deployment, enumerate their assets (secrets that are
-either inherently valuable or which confer some capability that enables further
-attack on the system), the capabilities that a malicious or compromised actor
-has, and potential mitigations for attacks enabled by those capabilities.
-
-This model assumes that all participants have previously agreed upon and
-exchanged all shared parameters over some unspecified secure channel.
-
-### Client/user
-
-#### Assets
-
-1. Unsharded measurements. Clients are the only actor that can ever see the
```

Comment:
Note to self: move assets to an appendix (unless I figure out a clever way to keep it in the main text).

Reply:
Looking back to this list of assets, all of the assets are enumerated in the list of attacks. I think it's more useful to specify the assets with the context of how they can be used/compromised in an attack.